src/Pagination/Pagination.css

```diff
@@ -3,4 +3,42 @@
     pointer-events: none;
     box-shadow: none;
     color: #999;
+}
+
+.pagination {
```

Comment:
Interesting! Pagination could be a cool feature. I'll need to look more into it.